### PR TITLE
FLU: Convert JetBot robot and JetsonNano device 

### PR DESCRIPTION
**Pull request by stefaniapedrazzi**
Convert:

robots:
* [x] (699) robots/nvidia/jetbot/protos/JetBotRaspberryPiCamera.proto
* [x] (700) robots/nvidia/jetbot/protos/JetBot.proto

worlds:
* [x] (255) projects/robots/nvidia/jetbot/worlds/jetbot.wbt

and devices:
* [x] projects/devices/nvidia/protos/JetsonNano.proto

### CODE REVIEW

**Comment by stefaniapedrazzi** [comment]
Windows test is failing because of Java and is not related to the changes in this PR.

**Comment by stefaniapedrazzi** [comment]
> Looks good! I have only one remark: the default rotation of the RaspberryPiCamera in the cameraSlot of the JetBot.proto seems wrong.

Not sure to understand what do you mean.
Do you refer to this line?
https://github.com/cyberbotics/webots/blob/d074158a2d161a879f3de837db607e746d032f98/projects/robots/nvidia/jetbot/worlds/jetbot.wbt#L261

To me it seems correct, both the object appearance and Camera image are oriented correctly.

The only thing is that effectively this object is not FLU (I will fix it) but this has nothing to do with the orientation in the `cameraSlot`.

**Comment by BenjaminDeleze** [comment]
> Do you refer to this line?

No to this one https://github.com/cyberbotics/webots/blob/d074158a2d161a879f3de837db607e746d032f98/projects/robots/nvidia/jetbot/protos/JetBot.proto#L22

**Comment by stefaniapedrazzi** [comment]
I also found errors in the `SolidRoundedBox` PROTO that is used in the JetBot robot.
This PROTO is also used in the a couple of `samples/devices` worlds that are not been yet converted and in these two worlds that are markes as converted but it seems they are still being processed:
* `projects/samples/environments/factory/worlds/hall.wbt` (I could not find the corresponding PR)
* `projects/samples/environments/factory/worlds/extended_solids.wbt` (see https://github.com/cyberbotics/webots/pull/3664, not merged)

File: projects/objects/solids/protos/SolidRoundedBox.proto [hunk 1]
```diff
@@ -48,6 +48,7 @@ PROTO SolidRoundedBox [
 
     const borderDiameter = 2 * borderRadius;
     const innerSize = {x: size.x - borderDiameter, y: size.y - borderDiameter, z: size.z - borderDiameter};
+    console.error('\'innerSize\'' + innerSize.x + ' '  + innerSize.y + ' '  + innerSize.z);
```
Comment (BenjaminDeleze) [inline projects/objects/solids/protos/SolidRoundedBox.proto:51]:
```suggestion
```
I think this is a debug log

**Review by BenjaminDeleze — CHANGES_REQUESTED** [review]
Thank you!
just one small thing:

File: projects/objects/solids/protos/SolidRoundedBox.proto (hunk 1 above)
Comment (stefaniapedrazzi) [reply projects/objects/solids/protos/SolidRoundedBox.proto]: Thank you!

**Review by BenjaminDeleze — APPROVED** [review]
Thank you!

**Pull request merged**